.gitignore

```diff
@@ -1,33 +1,34 @@
-*~
```

Comment:
Just have added `.vimrc` and sorted in alphabetical order, because there was no any structure before.